src/picongpu/include/simulation_defines/unitless/ionizerConfig.unitless

```diff
@@ -20,5 +20,6 @@
 
 #pragma once
 
+#include "particles/ionization/byField/ADK/AlgorithmADK.hpp"
```

Comment:
for those files, you can put it in a later PR.

for new namings that we marked, that are first-time introduced in this PR, please change them immediately, so there is no inconsistent naming in the meantime.